dwave/cloud/client.py

```diff
@@ -646,24 +651,62 @@ def solvers(self, name=None, qpu=None, software=None, vfyc=None,
             software (bool, default=None):
                 Should solver be software based?
 
-            vfyc (bool, default=None):
-                Should solver work on "virtual full-yield chip"?
-
-            flux_biases (bool, default=None):
-                Should solver accept flux biases?
-
-            anneal_schedule (bool, default=None):
-                Should solver accept anneal schedule?
-
-            num_qubits (int/[int,int], default=None):
-                What's the range (or exact number) of qubits solver should handle?
-
             online (bool, default=True):
                 Should solver be online?
 
             refresh (bool, default=False):
                 Force refresh cached list of solvers/properties
 
+        Filtering forms:
+            <parameter> (bool),
+            <parameter>__available (bool),
+            <parameter>__<operator> (object <value>):
+                Ensures solver supports `parameter`. General operator form can
+                be used, but that usually doesn't make sense for parameters,
+                since values are human-readable descriptions. Default operator
+                if `available`.
+
+            <property> (bool),
+            <property>__eq (bool),
+            <property>__<operator> (object <value>):
+                Ensures the value of solver's `property`, after applying
+                `operator` equals the righthand side `value`. Default operator
+                is `eq`.
+
+        Operators:
+            available, lt, lte, gt, gte, eq, within, contains, regex
+
+        Common solver parameters:
```

Comment:
Reversing roles here for a moment: why not save the docstring lines of these "Common solver parameters" and "Common solver properties", and let the examples, which use these same parameters/properties  speak for themselves?

Reply:
You're right, I need to slash all those examples a little bit :)